src/main/java/org/reactome/addlinks/EnsemblBiomartUtil.java

```diff
@@ -0,0 +1,148 @@
+package org.reactome.addlinks;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class EnsemblBiomartUtil {
+
+
+    public static final String proteinToGenesSuffix = "_proteinToGenes";
+    public static final String proteinToTranscriptsSuffix = "_proteinToTranscripts";
+    public static final String transcriptToMicroarraysSuffix = "_transcriptToMicroarrays";
+    public static final String uniprotToProteinsSuffix = "_uniprotToProteins";
+
+    /**
+     * Function that takes species name attribute from config file (eg: Homo sapiens) and modifies it to
+     * match Biomart formatting (first letter from primary species name + secondary name, all lowercase -- eg: hsapiens).
```

Comment:
Gotta keep my nitpick count respectable :) - You could say the first letter from the genus name and species name, all lowercase

Reply:
:roll_eyes:

Reply:
Ah, fair point!